source/widgets/ProjectStatus/configuration.html

```diff
@@ -15,10 +15,15 @@
                     <select id="octopus-space"></select>
                 </div>
             </div>
-            <div class="dropdown">
+            <div>
                 <label>Octopus Project</label>
                 <div class="wrapper">
-                    <select id="octopus-project"></select>
+                    <input type="text" id="octopus-project-filter" placeholder="Filter project by name (optional)" />
```

Comment:
Our configuration dashboard widget, this is how we inject a new project filter input box. It's not a dropdown, so it can't be wrapped in the "dropdown" css.